script/Ciel/Day 8/08_01_CIEL08_1.txt

```diff
@@ -351,7 +375,8 @@ There's probably not a single person in this city that can beat Arcueid.
 [sha:53a7298a0c5e97418080321352ad3b3bf220953e]{
 -- Page 26, Offset 18327.
 -- それではわたしも志貴さま同様、昨夜と同じ業務を行わせていただきます。どうか、お早いお帰りを」
-In that case, I shall conduct my duties as I did last night as well. I pray for your early return."
+In that case, I too shall act as I did last night.
```

Comment:
I don't really see the case for the simplification here.

>In that case, I too shall perform my duties as I did last night.

Might be a good middle ground.

Reply:
Felt a bit stiff, and I didn't like how it suggested that they were both doing _duties_, though yeah sure I guess you can consider what Shiki's up to a duty. Suggestion works for me.